### PR TITLE
Add list chart carousel MWPW-114550

### PR DESCRIPTION
**Pull request by Brandon32**
* Add carousel to list chart if more than one list.
* Fix lists not rendering if named sheet is missing.

Resolves: [MWPW-114550](https://jira.corp.adobe.com/browse/MWPW-114550)

**Test URLs:**
- Before: https://data-viz--milo--adobecom.hlx.page/drafts/bmarshal/list-chart
- After: https://list-carousel--milo--adobecom.hlx.page/drafts/bmarshal/list-chart

### CODE REVIEW

File: libs/blocks/chart/list.js [hunk 1]
```diff
@@ -39,12 +41,9 @@ export function listChartData(json) {
...
 
-export const getListHtml = (data) => {
-  // ToDo: Replace firstList with carousel iterator MWPW-114550
-  const firstList = data[0];
-  const listType = firstList.type?.toLowerCase() === 'numbered' ? 'ol' : 'ul';
-
-  const listItems = firstList.list.reduce((prev, { name = '', extra, image, alt = '' }) => (
+export const getListHtml = (list) => {
+  const listType = list.type?.toLowerCase() === 'numbered' ? 'ol' : 'ul';
+  const listItems = list.list.reduce((prev, { name = '', extra, image, alt = '' }) => (
```
Comment (meganthecoder) [inline libs/blocks/chart/list.js:46]: `list.list` is confusing. Perhaps the param should be `chart` or `data`

File: libs/blocks/chart/list.js [hunk 2]
```diff
@@ -55,19 +54,80 @@ export const getListHtml = (data) => {
...
+      </div>
+      <div id="listChartCarousel-items"
+        class="carousel-items"
+        aria-live="polite">
+        ${carouselItems}
+      </div>
+    </section>`;
+};
+
+const showCarouselItem = (element, index) => {
```
Comment (meganthecoder) [inline libs/blocks/chart/list.js:97]: Looks like you need a test for this one

**Review by meganthecoder — CHANGES_REQUESTED** [review]
Needs more tests. The diff coverage is only 60%. Other than that and one minor comment, looks good

File: libs/blocks/chart/list.js (hunk 2 above)
Comment (Brandon32) [reply libs/blocks/chart/list.js:97]: I was planning on adding this test after testing initList in MWPW-115223.

File: libs/blocks/chart/list.js (hunk 1 above)
Comment (Brandon32) [reply libs/blocks/chart/list.js]: `chart` sounds good, `data` could be confusing with the other `data`.

**Review by meganthecoder — APPROVED** [review]

**Pull request merged**